examples/computeToFandTracers.cpp

```diff
@@ -0,0 +1,195 @@
+/*
+  Copyright 2016 SINTEF ICT, Applied Mathematics.
+  Copyright 2016 Statoil ASA.
+
+  This file is part of the Open Porous Media Project (OPM).
+
+  OPM is free software: you can redistribute it and/or modify
+  it under the terms of the GNU General Public License as published by
+  the Free Software Foundation, either version 3 of the License, or
+  (at your option) any later version.
+
+  OPM is distributed in the hope that it will be useful,
+  but WITHOUT ANY WARRANTY; without even the implied warranty of
+  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+  GNU General Public License for more details.
+
+  You should have received a copy of the GNU General Public License
+  along with OPM.  If not, see <http://www.gnu.org/licenses/>.
+*/
+
+#if HAVE_CONFIG_H
+#include <config.h>
+#endif
+
+#include <opm/core/props/BlackoilPhases.hpp>
+
+#include <opm/flowdiagnostics/ConnectivityGraph.hpp>
+#include <opm/flowdiagnostics/ConnectionValues.hpp>
+#include <opm/flowdiagnostics/Toolbox.hpp>
+
+#include <opm/utility/ECLGraph.hpp>
+
+#include <exception>
+#include <iostream>
+#include <sstream>
+#include <stdexcept>
+#include <string>
+#include <utility>
+#include <vector>
+
+#include <boost/filesystem.hpp>
+
+namespace {
+    std::vector<std::string>
+    commandArguments(int argc, char* argv[])
+    {
+        return { argv, argc + argv };
+    }
+
+    bool isFile(const boost::filesystem::path& p)
+    {
+        namespace fs = boost::filesystem;
+
+        auto is_regular_file = [](const fs::path& pth)
+        {
+            return fs::exists(pth) && fs::is_regular_file(pth);
+        };
+
+        return is_regular_file(p)
+            || (fs::is_symlink(p) &&
+                is_regular_file(fs::read_symlink(p)));
+    }
+
+    boost::filesystem::path
+    deriveFileName(boost::filesystem::path         file,
+                   const std::vector<std::string>& extensions)
+    {
+        for (const auto& ext : extensions) {
+            file.replace_extension(ext);
+
+            if (isFile(file)) {
+                return file;
+            }
+        }
+
+        const auto prefix = file.parent_path() / file.stem();
+
```

Comment:
I think readability would be improved by increasing the blank line contrast: this part (lines 76 to 83) don't need blank lines in my opinion, while the space between functions (line 63) would be better with more blank lines.

Just a minor annoyance, not required for merging.